internal/github/action.go

```diff
@@ -106,7 +106,7 @@ func (a *Action) pullRequestNumber() int {
 }
 
 func (a *Action) resultStepOutput(result string) string {
-	return "::set-output name=label_check::" + result
+	return "echo 'label_check=result' >> $GITHUB_OUTPUT" 
```

Comment:
Your amendment is hardcoding the word `result` instead of interpolating the value of the `result` variable.  Should instead be:

`return fmt.Sprintf("echo 'label_check=%s' >> $GITHUB_OUTPUT", result)`